moi/test/test_job.py

```diff
@@ -48,19 +54,25 @@ def foo(a, b, **kwargs):
             return a+b
 
         r_client.set(self.test_job_info['id'], json.dumps(self.test_job_info))
-        _redis_wrap(self.test_job_info, foo, 1, 2)
+        obs_ret = _redis_wrap(self.test_job_info, foo, 1, 2)
 
-        sleep(2)
+        sleep(1)
         obs = json.loads(r_client.get(self.test_job_info['id']))
         self.assertEqual(obs['result'], 3)
+        self.assertEqual(obs_ret, obs['result'])
```

Comment:
`obs` is the result that is stored in redis, while `obs_ret` is the direct result of executing the method through `_redis_wrap`. This ensures that `_redis_wrap` is actually storing the expected result in redis.